defaults/main.yml

```diff
@@ -3,3 +3,4 @@ pecl_msgpack_version: 0.5.7
 pecl_zmq_version: 1.1.2
 
 php_fpm_daemon: php-fpm
+php_version: 'php5'
```

Comment:
better to check installed php version

Reply:
or if it's too complicated, we can have variables zeromq_install_from_source and msgpack_install_from_source